activesupport/lib/active_support/message_verifier.rb

```diff
@@ -166,6 +166,8 @@ class InvalidSignature < StandardError; end
     #   +config.active_support.use_message_serializer_for_metadata+.
     def initialize(secret, **options)
       raise ArgumentError, "Secret should not be nil." unless secret
+
+      options[:url_safe] = true if options[:url_safe].nil?
```

Comment:
This would be a breaking change.  Why change it?

Reply:
Due to [[this change](https://github.com/rails/rails/pull/49507)](https://github.com/rails/rails/pull/49507), `signed_id` needs to be generated in a URL-safe manner. Since this configuration applies to all message verifiers, I couldn't find a way other than changing this. However, I acknowledge that this could be a breaking change.